scripts/build_third_party_licenses.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# A script to generate a THIRD_PARTY_LICENSES file containing all the licenses that we use from third parties.
+# This is intended to be run from the project root as `$ ./scripts/build_third_party_licenses.sh`
+# NOTE: This only adds licenses from go dependencies. For other licenses, see NOTICE.md
+
+# The apache 2.0 license doesn't get modified with a copywrite. To reduce duplication, add attribution for each project using the license, but include the license text just once.
+go-licenses report --template=scripts/third_party_licenses/apache.tpl --ignore github.com/awslabs/soci ./... > ./THIRD_PARTY_LICENSES
```

Comment:
Is it expected to produce the `THIRD_PARTY_LICENSES` file in whichever location the script being run (e.g. `./random-dir/soci-snapshotter/scripts/build_third_party_licenses.sh` will produce the file under `random-dir`)? or it's expected to be produced at the soci root?

If it's the latter, suggest to obtain soci root dir like below and redirect the file to there:

https://github.com/awslabs/soci-snapshotter/blob/fb32f8751bc5d0810c595da9936ac63a8a7ff336/scripts/check-ltag.sh#L19-L20

Reply:
Yeah, there's a comment here that it assumes it's run from the project root directory. I can switch to match `check-ltag.sh`.